### PR TITLE
Update instructions.md

### PR DESCRIPTION
**Pull request by dsletten**
The inclusion of the word "positive" is redundant given the context. Moreover, it is inconsistent with the test suite. The test suite expects a result of 0.5 for a balance of 0, which is not a positive number.

### CODE REVIEW

File: exercises/concept/interest-is-interesting/.docs/instructions.md [hunk 1]
```diff
@@ -3,9 +3,9 @@
 In this exercise you'll be working with savings accounts. Each year, the balance of your savings account is updated based on its interest rate. The interest rate your bank gives you depends on the amount of money in your account (its balance):
 
 - -3.213% for a negative balance.
-- 0.5% for a positive balance less than `1000` dollars.
-- 1.621% for a positive balance greater or equal than `1000` dollars and less than `5000` dollars.
-- 2.475% for a positive balance greater or equal than `5000` dollars.
+- 0.5% for a balance less than `1000` dollars.
+- 1.621% for a balance greater or equal than `1000` dollars and less than `5000` dollars.
```
Comment (BTowersCoding) [inline exercises/concept/interest-is-interesting/.docs/instructions.md:7]: Thanks! While taking a closer look I happened to notice that the wording here is strange, maybe we could fix this now as well?
```suggestion
- 1.621% for a balance greater than or equal to `1000` dollars and less than `5000` dollars.
```

File: exercises/concept/interest-is-interesting/.docs/instructions.md [hunk 2]
```diff
@@ -3,9 +3,9 @@
...
+- 2.475% for a balance greater or equal than `5000` dollars.
```
Comment (BTowersCoding) [inline exercises/concept/interest-is-interesting/.docs/instructions.md:8]: Also here:
```suggestion
- 2.475% for a balance greater than or equal to `5000` dollars.
```

**Review by bobbicodes — DISMISSED** [review]

File: exercises/concept/interest-is-interesting/.docs/instructions.md (hunk 2 above)
Comment (dsletten) [reply exercises/concept/interest-is-interesting/.docs/instructions.md:8]: I missed that. Your suggestion reads more naturally.

**Comment by tasxatzial** [comment]
This PR is stale at this point and conflicts with a very recent commit that already applied Bobbi's suggestions. I'm going to resolve the conflicts and then merge, provided no further changes are necessary.

**Review by tasxatzial — APPROVED** [review]

**Pull request merged**